requirements/deploy.txt

```diff
@@ -353,7 +353,7 @@ s3transfer==0.10.1
     #   boto3
 selectolax==0.3.21
     # via -r requirements/pip.txt
-sentry-sdk==1.45.0
+sentry-sdk==2.0.1
```

Comment:
We should double check this upgrade with the changelog.

Reply:
https://docs.sentry.io/platforms/python/migration/1.x-to-2.x

Also, this looks like a dependency from another package, hmm. We should probably track this one as a proper dependency.

Reply:
Cool! I don't see anything affecting us in that migration guide. I think we will be fine here.

Reply:
Even checking the underlying code seems there are no impactful changes: https://github.com/kiwicom/structlog-sentry/blob/master/structlog_sentry/__init__.py